src/features/user/MFV2/ProfileCard/ProfileCardButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import styles from './ProfileCardButton.module.scss';
+import Link from 'next/link';
+
+interface CommonProps {
+  icon?: ReactElement;
+  text?: string;
+  type?: 'primary' | 'secondary';
```

Comment:
Minor change - I suggest calling this `color` as it indicates the purpose better.